moto/rds/models.py

```diff
@@ -2309,8 +2332,9 @@ def get_snapshot(
         self,
         identifier: str,
         resource_type: type[DBSnapshot] | type[DBClusterSnapshot],
-        not_found_exception: type[DBSnapshotNotFoundFault]
-        | type[DBClusterSnapshotNotFoundError],
+        not_found_exception: (
+            type[DBSnapshotNotFoundFault] | type[DBClusterSnapshotNotFoundError]
+        ),
```

Comment:
This seems unrelated to your changeset.  If reverted, it would make this PR purely additive (all green) which would be ideal!